frontend/src/static/js/v_ramp.js

```diff
@@ -57,6 +36,42 @@ window.vRamp = {
       const newSize = 100 * (slice.insertions / this.avgsize);
       return Math.max(newSize * this.rampSize, 0.5);
     },
+    getTitle(slice, commit) {
+      let title = '';
+      if (this.tframe === 'commit') {
+        return `[${slice.date}] ${commit.messageTitle}: ${commit.insertions} lines`;
+      }
+
+      if (this.zoomfromramp) {
```

Comment:
This is not necessary as all ramp slice when zoom from ramp will be of granularity commit

Reply:
No the tframe will still be week/day accordingly. That's why I need the zoomfromramp as the title of ramp will be slightly different. If it is zoomfromramp and with week granularity the title will be for eg `week of 2018-06-04 [#11] Implement logger`. If it is not from ramp and commit then will be `[2018-06-04] [#11] Implement logger`.

Reply:
Currently, when clicking on ramps with granularity `day` or `week`, it shows the zoom tab with a ramp chart consists of ramps with granularity `commit`. 

Instead of showing `week of 2018-06-04 [#11] Implement logger` as the title, standardising it with normal ramps with granularity commits (i.e. displays the true commit date) would be a better behaviour.